### DISCUSSION TITLE
Upgrade guide

### DISCUSSION THREAD

**Discussion by IljaMartinovs** [Power-Components/livewire-powergrid#1208] [Q&A] [answered]
Hi. Can you please provide upgrade guide from v3 to v5 or v4. I found only from v4 to v5. Thanks!

**Comment by luanfreitasdev** [comment, marked as answer]
https://github.com/Power-Components/powergrid-doc/blob/3x/docs/get-started/upgrade-guide.md

v3 -> v4
v4 -> v5

**Reply by IljaMartinovs** [reply]
@luanfreitasdev Thanks a lot!
